pkg/asset-manager-utils/test/AaveATokenAssetManager.test.ts

```diff
@@ -154,33 +154,66 @@ describe('Aave Asset manager', function () {
-      const updatedConfig = { targetPercentage: 3, criticalPercentage: 2, feePercentage: 1 };
+      const updatedConfig = {
+        targetPercentage: 3,
+        upperCriticalPercentage: 4,
+        lowerCriticalPercentage: 2,
+        feePercentage: 1,
+      };
       await assetManager.connect(poolController).setPoolConfig(poolId, updatedConfig);
 
       const result = await assetManager.getPoolConfig(poolId);
       expect(result.targetPercentage).to.equal(updatedConfig.targetPercentage);
-      expect(result.criticalPercentage).to.equal(updatedConfig.criticalPercentage);
+      expect(result.upperCriticalPercentage).to.equal(updatedConfig.upperCriticalPercentage);
+      expect(result.lowerCriticalPercentage).to.equal(updatedConfig.lowerCriticalPercentage);
       expect(result.feePercentage).to.equal(updatedConfig.feePercentage);
     });
 
-    it('reverts when setting target over 100%', async () => {
-      const badPoolConfig = { targetPercentage: fp(1.1), criticalPercentage: 0, feePercentage: 0 };
+    it('reverts when setting upper critical over 100%', async () => {
+      const badPoolConfig = {
+        targetPercentage: 0,
+        upperCriticalPercentage: fp(1.1),
+        lowerCriticalPercentage: 0,
+        feePercentage: 0,
+      };
       await expect(assetManager.connect(poolController).setPoolConfig(poolId, badPoolConfig)).to.be.revertedWith(
-        'Investment target must be less than or equal to 100%'
+        'Upper critical level must be less than or equal to 100%'
       );
     });
 
-    it('reverts when setting critical above target', async () => {
-      const badPoolConfig = { targetPercentage: 1, criticalPercentage: 2, feePercentage: 0 };
+    it('reverts when setting upper critical below target', async () => {
+      const badPoolConfig = {
+        targetPercentage: 1,
+        upperCriticalPercentage: 0,
+        lowerCriticalPercentage: 0,
+        feePercentage: 0,
+      };
       await expect(assetManager.connect(poolController).setPoolConfig(poolId, badPoolConfig)).to.be.revertedWith(
-        'Critical level must be less than target'
+        'Target must be less than or equal to upper critical level'
+      );
+    });
+
+    it('reverts when setting lower critical above target', async () => {
+      const badPoolConfig = {
+        targetPercentage: 1,
+        upperCriticalPercentage: 2,
+        lowerCriticalPercentage: 2,
+        feePercentage: 0,
+      };
+      await expect(assetManager.connect(poolController).setPoolConfig(poolId, badPoolConfig)).to.be.revertedWith(
+        'Lower critical level must be less than or equal to target'
       );
     });
 
     it('reverts when setting fee percentage over 100%', async () => {
-      const badPoolConfig = { targetPercentage: 0, criticalPercentage: 0, feePercentage: fp(1.01) };
+      const badPoolConfig = {
+        targetPercentage: 0,
+        upperCriticalPercentage: 0,
+        lowerCriticalPercentage: 0,
+        feePercentage: fp(1.01),
```

Comment:
```suggestion
        feePercentage: fp(0.1).add(1),
```

1.01 would be a fee percentage of 101%, not 10%